src/views/Guide.vue

```diff
@@ -29,20 +29,29 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     -->
 
     <h2 id="task-and-job-states">
-      Task And Job States
+      Understanding Tasks &amp; Jobs
     </h2>
-    <!-- TODO document difference between tasks and jobs -->
+
+    <p>
+      <b>Tasks</b> represent single units of activity in a Cylc workflow.<br/>
+      <b>Jobs</b> are real processes submitted by Cylc to perform a task's activity.<br/>
+      One task can have multiple jobs due to automatic retries or manual triggering.
+    </p>
+
     <div class="card-grid">
 
       <v-flex
         md6
         xs12
       >
-        <v-card>
+        <v-card outlined>
           <v-card-title primary-title>
-            <p class="display-1 text--primary">Task State Vs Job State</p>
+            <p class="display-1 text--primary">Task &amp; Job States</p>
           </v-card-title>
+
           <v-card-text>
+            Tasks represent future and past jobs as well as current jobs.<br/>
+            A <b>waiting</b> task that already has job(s) will <b>retry</b>.
```

Comment:
The spacing after this text is a bit short. I think it either needs to be wrapped in a `<p>` or followed by a `<br />`.

Reply:
Done, it looks OK now (screenshot not updated).